docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -110,13 +111,13 @@ Refer to the [advanced Identity configuration guide](./identity.md) for addition
 Web Modeler uses the [Zeebe Java client](/apis-tools/java-client/index.md) to connect to Zeebe.
 To customize the client configuration, you can provide optional environment variables.
 
-| Environment variable          | Description                                                                                              | Example value                    | Default Value                |
-| ----------------------------- | -------------------------------------------------------------------------------------------------------- | -------------------------------- | ---------------------------- |
-| `ZEEBE_CA_CERTIFICATE_PATH`   | [optional]<br/>Path to a root CA certificate to be used instead of the certificate in the default store. | `/path/to/certificate`           | -                            |
-| `ZEEBE_CLIENT_CONFIG_PATH`    | [optional]<br/>Path to the client's OAuth credential cache.                                              | `/path/to/credentials/cache.txt` | `$HOME/.camunda/credentials` |
-| `ZEEBE_CLIENT_REQUESTTIMEOUT` | [optional]<br/>The request timeout used when communicating with a target Zeebe cluster.                  | `60000`                          | `10000`                      |
-| `ZEEBE_AUTH_CONNECT_TIMEOUT`  | [optional]<br/>The connection timeout for requests to the OAuth server.                                  | `30000`                          | `5000`                       |
-| `ZEEBE_AUTH_READ_TIMEOUT`     | [optional]<br/>The data read timeout for requests to the OAuth server.                                   | `30000`                          | `5000`                       |
+| Environment variable            | Description                                                                                              | Example value                    | Default Value                |
+| ------------------------------- | -------------------------------------------------------------------------------------------------------- | -------------------------------- | ---------------------------- |
+| `CAMUNDA_CA_CERTIFICATE_PATH`   | [optional]<br/>Path to a root CA certificate to be used instead of the certificate in the default store. | `/path/to/certificate`           | -                            |
```

Comment:
Adding the comment here, because GitHub does not let me add it on the correct lines above: Let's also rename "Zeebe client" and "Zeebe Java client" to "Camunda client" and "Camunda Java client".